src/middleware/validation.middleware.ts

```diff
@@ -55,6 +58,26 @@ export function checkValidations(req: Request, res: Response, next: NextFunction
       }
       const entityNumber = appData?.[EntityNumberKey];
 
+      if (isActiveFeature(config.FEATURE_FLAG_ENABLE_REDIS_REMOVAL)) {
+        return res.render(NAVIGATION[routePath].currentPage, {
+          backLinkUrl: NAVIGATION[routePath].previousPage(appData, req),
+          templateName: NAVIGATION[routePath].currentPage,
+          id,
+          entityName,
+          entityNumber,
+          ...appData,
+          ...req.body,
+          ...dates,
+          errors,
+          FEATURE_FLAG_ENABLE_REDIS_REMOVAL: true,
```

Comment:
I feel there should be a comment here explaining why this block of code is here. It's being done for one specific screen (at present) and as discussed, not the nicest of solutions.

Happy with this solution for now, but if we ever have another screen that requires more such links to be passed in, that would be the time to refactor...

Reply:
added a comment